README.md

```diff
@@ -1,15 +1,96 @@
+| [stripe__churned_customers_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__churned_customers_monthly.sql)    | Monthly count of customers that churned.
+| [stripe__mrr](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__mrr.sql)    | Monthly recurring revenue.
+| [stripe__new_customer_revenue_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__new_customers_monthly.sql)    |         Monthly revenue due to customers that signed up for the first time.                                                               |
+| [stripe__new_customers_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__churned_customers_monthly.sql)    |  Monthly count of customers that signed up for the first time.                                     |
+| [stripe__platform_fees_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__platform_fees_monthly.sql)    | Monthly fees paid to Stripe.                                                         |
+| [stripe__recovered_customer_revenue_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__recovered_customer_revenue_monthly.sql)    | Monthly revenue due recovery of previously churned customers.               |
+| [stripe__recovered_customers_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__recovered_customers_monthly.sql)    | Monthly number of previously churned customers who signed up again.|                                                                                                                                 
+
+# 🎯 How do I use the dbt package?
+## Step 1: Prerequisites
+To use this dbt package, you must have the following:
+- At least one Fivetran stripe connector syncing data into your destination. 
+- A **BigQuery**, **Snowflake**, **Redshift**, or **PostgreSQL** destination.
+
+
+## Step 2: Install the package
+
+Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+
+Include in your `packages.yml`
+
+```yaml
+packages:
+  - git: "git@github.com:HousewareHQ/dbt_stripe_metrics.git"
+    revision: 0.1.0
+```
+
+## Step 3: Define database and schema variables
+
+By default, this package will look for your Stripe data in the `stripe` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Stripe data is, please add the following configuration to your `dbt_project.yml` file:
```

Comment:
Let's either update the variable `stripe_schema` in dbt_project.yml to `stripe` or the update here as `fivetran_stripe`

Reply:
https://github.com/HousewareHQ/dbt_stripe_metrics/pull/5#discussion_r905977780